core-web/libs/edit-content/src/lib/fields/dot-edit-content-wysiwyg-field/dot-edit-content-wysiwyg-field.component.ts

```diff
@@ -1,37 +1,58 @@
 import { EditorModule, TINYMCE_SCRIPT_SRC } from '@tinymce/tinymce-angular';
-import { RawEditorOptions } from 'tinymce';
+import { RawEditorOptions, TinyMCE } from 'tinymce';
 
 import { ChangeDetectionStrategy, Component, Input, inject, signal } from '@angular/core';
 import { ControlContainer, FormsModule, ReactiveFormsModule } from '@angular/forms';
 
+import { DialogService } from 'primeng/dynamicdialog';
+
 import { DotCMSContentTypeField } from '@dotcms/dotcms-models';
 
+import { DotWysiwygPluginService } from './dot-wysiwyg-plugin/dot-wysiwyg-plugin.service';
+
+declare global {
+    interface Window {
+        tinymce: TinyMCE;
+    }
+}
+
 @Component({
     selector: 'dot-wysiwyg-field',
     standalone: true,
     imports: [EditorModule, FormsModule, ReactiveFormsModule],
-    templateUrl: './dot-wysiwyg-field.component.html',
-    styleUrl: './dot-wysiwyg-field.component.scss',
+    templateUrl: './dot-edit-content-wysiwyg-field.component.html',
+    styleUrl: './dot-edit-content-wysiwyg-field.component.scss',
     changeDetection: ChangeDetectionStrategy.OnPush,
-    providers: [{ provide: TINYMCE_SCRIPT_SRC, useValue: 'tinymce/tinymce.min.js' }],
+    providers: [
+        DialogService,
+        DotWysiwygPluginService,
+        { provide: TINYMCE_SCRIPT_SRC, useValue: 'tinymce/tinymce.min.js' }
```

Comment:
TIL! How this works?

Reply:
I followed their [documentation](https://www.tiny.cloud/docs/tinymce/latest/angular-ref/#installing-the-tinymce-angular-integration-using-npm).

In summary, `TINYMCE_SCRIPT_SRC` is an injection token pointing to their `cloud`. To use the `MIT` license, we must direct that source to our local `script` installed by `NPM`.